src/sagas/bucket.js

```diff
@@ -50,11 +50,11 @@ export function* updateBucket(getState, action) {
 }
 
 export function* deleteBucket(getState, action) {
-  const { bid } = action;
+  const { bid, last_modified } = action;
```

Comment:
Note that from here, current bucket last_modified could be easily retrieved from `getState().bucket.data`.